app/models/form.rb

```diff
@@ -35,7 +35,9 @@ def subsection_for_page(page)
   def next_page(page, case_log)
     page_ids = subsection_for_page(page).pages.map(&:id)
     page_index = page_ids.index(page.id)
-    nxt_page = get_page(page_ids[page_index + 1])
+    page_id = page.id.include?("value_check") && case_log[page.id] == 1 ? page_ids[page_index - 1] : page_ids[page_index + 1]
```

Comment:
Is there a test for this specific logic? i.e. one asserting what should be returned for each of the cases?

Also when is `case_log[page.id] == 1` true? What is 1 there?

Reply:
This is my bad I should have added a comment here! `case_log[page.id]` in this case becomes `case_log[net_income_value_check]` and the 1 refers to choosing no on the radio question

Reply:
That makes sense! Might be worth extracting a method to make that a little more obvious.

Reply:
But the idea is that any interruption screen page will have a page name containing `value_check` and the accompanying radio question will also have `value_check` in the name so `case_log[page.id]` will become `case_log[*field_name*_value_check]`

Reply:
Hopefully I've explained that clearly! 😅